screen.go

```diff
@@ -6,41 +6,52 @@ import (
 )
 
 const (
-	AltScreenSeq             = "?1049h"
-	ExitAltScreenSeq         = "?1049l"
-	CursorUpSeq              = "%dA"
-	CursorDownSeq            = "%dB"
-	CursorForwardSeq         = "%dC"
-	CursorBackSeq            = "%dD"
-	CursorNextLineSeq        = "%dE"
-	CursorPreviousLineSeq    = "%dF"
-	CursorHorizontalSeq      = "%dG"
-	CursorPositionSeq        = "%d;%dH"
-	EraseDisplaySeq          = "%dJ"
-	EraseLineSeq             = "%dK"
-	ScrollUpSeq              = "%dS"
-	ScrollDownSeq            = "%dT"
-	SaveCursorPositionSeq    = "s"
-	RestoreCursorPositionSeq = "u"
-	ShowCursorSeq            = "?25h"
-	HideCursorSeq            = "?25l"
-	ChangeScrollingRegionSeq = "%d;%dr"
-	InsertLineSeq            = "%dL"
-	DeleteLineSeq            = "%dM"
+	CursorUpSeq               = "%dA"
+	CursorDownSeq             = "%dB"
+	CursorForwardSeq          = "%dC"
+	CursorBackSeq             = "%dD"
+	CursorNextLineSeq         = "%dE"
+	CursorPreviousLineSeq     = "%dF"
+	CursorHorizontalSeq       = "%dG"
+	CursorPositionSeq         = "%d;%dH"
+	EraseDisplaySeq           = "%dJ"
+	EraseLineSeq              = "%dK"
+	ScrollUpSeq               = "%dS"
+	ScrollDownSeq             = "%dT"
+	SaveCursorPositionSeq     = "s"
+	RestoreCursorPositionSeq  = "u"
+	ChangeScrollingRegionSeq  = "%d;%dr"
+	InsertLineSeq             = "%dL"
+	DeleteLineSeq             = "%dM"
+	ShowCursorSeq             = "?25h"
```

Comment:
We should add an empty line between the two sections of sequences here, to prevent code-formatting from resorting it.